### ISSUE TITLE
Can't import the named export 'ScrollMenu'

### ISSUE THREAD

**Issue by ShumiGupta** [asmyshlyaev177/react-horizontal-scrolling-menu#234]
Hi! 👋

Firstly, thanks for great library! 🙂

We are importing react-horizontal-scrolling-menu in our React App. While using **v4.0.3** I see below error where as everything seems to work fine with **v4.0.1**

**React version**: 18.2.0

**Import Code:** `import { ScrollMenu } from "react-horizontal-scrolling-menu";`

**Error:** `Can't import the named export 'ScrollMenu' (imported as 'ScrollMenu') from default-exporting module (only default export is available)`

**Comment by asmyshlyaev177** [comment]
Hi, can you please post more info about your environment, e.g. CRA/NextJS|webpack and version.

**Comment by barry-genies** [comment]
I also meet the same issue even if my `react-horizontal-scrolling-menu` is 4.0.1.  but import VisibilityContext works. I just update many dependencies under my project so I guess it's because of one of them...
some of dependency version in my project:
```
"typescript": "^4.9.3",
"react-scripts": "^5.0.1",
"eslint": "^8.15.0",
"react": "^18.2.0",
"react-dom": "^18.2.0",
"webpack": "5.79.0"
```

can be solved through:
```
import { VisibilityContext } from "react-horizontal-scrolling-menu";
import Menu from "react-horizontal-scrolling-menu";

<Menu.ScrollMenu onWheel={onWheel}>
    {children}
</Menu.ScrollMenu>
```

but if I try this:`import * as Menu from "react-horizontal-scrolling-menu";`, console will show error:` Can't import the named export 'ScrollMenu' (imported as 'Menu') from default-exporting module (only default export is available)`

the print result of Menu is:
`/static/media/index.e9788367c27c37b8cbd5.cjs`

**Comment by asmyshlyaev177** [comment]
> I'm having a weird issue where my scroll isn't being implemented - I've bypassed the bug of the can't import the named export...but my scroll is just not working horizontally for some reason?
> 
> I've tried importing the CSS you recommended, but when I do that it cuts off and only shows a portion of the day that I want to see and isn't scrollable

Let's keep it 1 issue 1 problem, and better to have examples to reproduce bugs.

**Comment by percy507** [comment]
Same problem. I am using vite@2.9.5.

```bash
# error log
Uncaught SyntaxError: The requested module '/node_modules/.pnpm/react-horizontal-scrolling-menu@4.0.3_react@18.2.0/node_modules/react-horizontal-scrolling-menu/dist/index.cjs?import' does not provide an export named 'ScrollMenu' 
```

**Comment by asmyshlyaev177** [comment]
Released `4.0.4` , check if it works properly.

**Comment by barry-genies** [comment]
> Released `4.0.4` , check if it works properly.

upgrading to 4.0.4 works for me:) thx for quick fixing
